docs/io/visualization/how_to_liv_plot.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>What is the y-range when plotting in log-space?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>